device/dell/x86_64-dellemc_z9264f_c3538-r0/plugins/psuutil.py

```diff
@@ -35,8 +37,10 @@ def get_pmc_register(self, reg_name):
         status = 1
         global ipmi_sdr_list
         ipmi_dev_node = "/dev/pmi0"
-
-        ipmi_cmd = IPMI_SENSOR_DATA
+        ipmi_cmd = IPMI_PSU_DATA
+        
+        if os.path.exists("/.dockerenv"):
```

Comment:
Apparently this is not a good long-term solution to check for whether we are running in a Docker container. Docker also used to create a `./dockerinit` file, but they have since removed it. There's a chance they remove `/.dockerenv` in the future, as well. It seems that a better solution is something is something like the following (in bash):
```
if [ $(cat /proc/self/cgroup | grep -c ":/docker") -gt 0 ] ; then
    echo "Running in a Docker container"
else
    echo "NOT running in a Docker container"
fi
```

Reply:
Thanks for your valuable feedback,
As per you suggestion changes are updated and tested. 
Can you please review.